Docs release 1.10.0 csi 4365 host define (#517)
IBM/ibm-block-csi-driver: docs/content/installation/upgrade_manual_openshift.md

12. Check the **Overview** tab and that the **Controller Image Tab** and **Node Image Tag** are showing the most up-to-date version of the driver and the **Status** is _Running_.

> 13. (Optional for initial host definer installation) If desired, create the host definer by clicking **Create Instance** to create the host definer (`IBMBlockCSI_hostdefiner`).

Reviewer: @matancarmeli7 is `IBMBlockCSI_hostdefiner` also in the latest web console?

Reviewer 2: I don't know what is in the create instance page (I don't think the there is `IBMBlockCSI_hostdefiner`)

Author: I think that I had put this in at one point as a placeholder and it got through approvals.
I'll update to `HostDefiner` as it is listed in the installation section (which was taken from screen captures)

Reviewer: @rivkap could you revisit all the web console steps with someone from QA?
I suggest you do a live walkthrough session together. otherwise it's hard to see the process with still screenshots

Author: updated

Reviewer: @rivkap this is about more than the `IBMBlockCSI_hostdefiner` placeholder (e.g. it seems there is no such word as "instance" there).
please see a live/recorded walkthrough, to make sure nothing else is missed

Reviewer: and regardless, in general, please use more explicit placeholders (e.g. `<"something"-placeholder>`), so we could tell them apart from official content more easily

Author: Updated all installation / upgrade/ uninstall sections to do with web console
@oriyarde please resolve if possible. thanks

Reviewer: LGTM. thanks!

> 13. (Optional for initial installation during upgrade only) If desired, create the host definer by clicking click **Create Instance** to create the host definer (`IBMBlockCSI_hostdefiner`).

Reviewer: ```suggestion
13. (Optional for initial installation during upgrade only) If desired, create the host definer by clicking **Create Instance** to create the host definer (`IBMBlockCSI_hostdefiner`).
```

Author: updated
note - "IBMBlockCSI_hostdefiner" is just a placeholder.
waiting on @matancarmeli7  for final name

> 13. (Optional for initial installation during upgrade only) If desired, create the host definer by clicking click **Create Instance** to create the host definer (`IBMBlockCSI_hostdefiner`).

Reviewer: ```suggestion
13. (Optional, if wasn't done before) If desired, create the host definer by clicking click **Create Instance** to create the host definer (`IBMBlockCSI_hostdefiner`).
```
not sure what `initial installation during upgrade only` means - creating the host definer is not for upgrade only, and generally speaking, "upgrade" is not an "initial installation"

Author: while upgrade for the _driver_ isn't an initial installation, if someone is upgrading this may be an initial installation for the host definer, especially for 1.10.

text updated
